### PR TITLE
Added ruleset for bundesfinanzhof.de

### PR DESCRIPTION
**Pull request by Jochen-A-Fuerbacher**

### CODE REVIEW

File: src/chrome/content/rules/Bundesfinanzhof.de.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<ruleset name="Bundesfinanzhof.de">
+	<!-- Federal Fiscal Court -->
+
+	<target host="bundesfinanzhof.de" />
+	<target host="www.bundesfinanzhof.de" />
+	<target host="juris.bundesfinanzhof.de" />
+
```
Comment (YegorIevlev) [inline src/chrome/content/rules/Bundesfinanzhof.de.xml]:
```xml
<securecookie host=".+" name=".+" />
```

File: src/chrome/content/rules/Bundesfinanzhof.de.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<ruleset name="Bundesfinanzhof.de">
+	<!-- Federal Fiscal Court -->
```
Comment (YegorIevlev) [inline src/chrome/content/rules/Bundesfinanzhof.de.xml]: Move this to the beginning of the file.

File: src/chrome/content/rules/Bundesfinanzhof.de.xml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<!-- Federal Fiscal Court -->
+<ruleset name="Bundesfinanzhof.de">
+
+	<target host="bundesfinanzhof.de" />
+	<target host="www.bundesfinanzhof.de" />
+	<target host="juris.bundesfinanzhof.de" />
+
+	<securecookie host=".+" name=".+" />
+
+	<!-- Invalid certificate on https://bundesfinanzhof.de/,
```
Comment (YegorIevlev) [inline src/chrome/content/rules/Bundesfinanzhof.de.xml]: Works for me.

Comment (Jochen-A-Fuerbacher) [reply src/chrome/content/rules/Bundesfinanzhof.de.xml]: The cert is only valid for `www.bundesfinanzhof.de`. On Firefox I get `SSL_ERROR_BAD_CERT_DOMAIN`.

**Pull request merged**